microsoft-365/security/office-365-security/whats-new-in-defender-for-office-365.md

```diff
@@ -38,6 +38,10 @@ Learn more by watching [this video](https://www.youtube.com/watch?v=Tdz6KfruDGo&
 > [!TIP]
 > Don't have Microsoft Defender for Office 365 yet? [Contact sales to start a trial](https://info.microsoft.com/ww-landing-M365SMB-web-contact.html).
 
+## August 2021
+
+- [Admin review for reported messages](admin-review-reported-message.md): admins can now send templated messages back to end users after they review reported messages. This can be customized for your organization and based on your admin's verdict as well. 
```

Comment:
```suggestion
- [Admin review for reported messages](admin-review-reported-message.md): Admins can now send templated messages back to end users after they review reported messages. This can be customized for your organization and based on your admin's verdict as well. 
```